src/ray/raylet/scheduling_queue.cc

```diff
@@ -0,0 +1,101 @@
+#ifndef SCHEDULING_QUEUE_CC
+#define SCHEDULING_QUEUE_CC
+
+#include "scheduling_queue.h"
+
+#include <list>
+
+#include "ray/status.h"
+#include "ray/id.h"
+
+namespace ray {
+
+const std::list<Task> &SchedulingQueue::waiting_tasks() const {
+  return this->waiting_tasks_;
+}
+
+const std::list<Task> &SchedulingQueue::ready_tasks() const {
+  return this->ready_tasks_;
+}
+
+const std::list<Task> &SchedulingQueue::scheduled_tasks() const {
+  return this->scheduled_tasks_;
+}
+
+const std::list<Task> &SchedulingQueue::running_tasks() const {
+  return this->running_tasks_;
+}
+
+const std::list<Task>& SchedulingQueue::ready_methods() const {
+  throw std::runtime_error("Method not implemented");
```

Comment:
No exception please

Reply:
Modern C++ doesn't bans exceptions I think.

Reply:
This is a placeholder to be replaced with implementation after the merge. As a reminder, the major goal of this initial PR was to get to the point where multiple people can contribute implementation, but setting most of the scaffolding in place...